largescale/many-collection-test.py

```diff
@@ -448,8 +448,9 @@ def launch_server_status_processor(name, conf):
     worst_reads_latency_value = 0
     worst_writes_latency_value = 0
     print_msg("ServerStat", 0, "   At | Five worsts over whole run")
```

Comment:
I was trying to keep this list to 5 worsts. Is the intention here to show atleast all the stalls ?

Reply:
`min` is used, it will be maximum the 5 worst. I did this when the scenario would be too short so less 5 samples were available.